vagrant/ansible/roles/samba-glusterfs.setup/templates/smb.conf.j2

```diff
@@ -1,19 +1,13 @@
 [global]
+workgroup = MYGROUP
+server string = Samba Server Version %v
+log file = /var/log/samba/log.%m
+max log size = 50
+security = user
+passdb backend = tdbsam
+load printers = yes
```

Comment:
I don't think we want clustered printing... ;-)

FWIW, we want the options from the original file, that essentially disable printing. Ok, this is done in the include file, but why bother mentioning these options here at all if they are being overwritten anyway?

Reply:
Ah..my bad.
I am merging both into smb.conf.j2.